scripts/src/test/org/oppia/android/scripts/RegexPatternValidationCheckTest.kt

```diff
@@ -0,0 +1,133 @@
+package org.oppia.android.scripts
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+/** Tests for [RegexPatternValidationCheck]. */
+class RegexPatternValidationCheckTest {
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = java.lang.System.out
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Before
+  fun setUpTests() {
+    tempFolder.newFolder("testfiles")
+    java.lang.System.setOut(PrintStream(outContent))
```

Comment:
Here & below--can we not just reference System directly? Why does it need to be fully qualified?

Reply:
Removed. Apologies! as this was added by the auto competition of Android studio.